public/grpc-client.js

```diff
@@ -93,6 +96,22 @@ module.exports.init = async function({
     });
   });
 
+  ipcMain.on('lndAtplInit', async event => {
+    credentials = await getCredentials(lndSettingsDir);
```

Comment:
Any reason we need to reload the credentials? Why not use the one already loaded above?